drivers/sensor/ina23x/ina237.c

```diff
@@ -63,7 +63,7 @@ static int ina237_channel_get(const struct device *dev, enum sensor_channel chan
 
 	case SENSOR_CHAN_POWER:
 		/* see datasheet "Current and Power calculations" section */
-		power_uw = (data->power * INA237_POWER_SCALING * config->current_lsb) / 10000U;
+		power_uw = (data->power * INA237_POWER_SCALING * config->current_lsb) / 10U;
```

Comment:
Since data->power is 24 bits and config->current_lsb can be up to 16-bits, this will still overflow for larger current LSBs.  Will add a test and update this today.

Reply:
Fixed and added two new DT test configurations to catch this overflow.